japicmp/src/main/java/japicmp/compat/CompatibilityChanges.java

```diff
@@ -178,8 +178,12 @@ public Integer callback(JApiClass superclass, Map<String, JApiClass> classMap, J
 					addCompatibilityChange(field, JApiCompatibilityChange.FIELD_NO_LONGER_STATIC);
 				}
 			}
-			if (isNotPrivate(field) && field.getType().hasChanged()) {
-				addCompatibilityChange(field, JApiCompatibilityChange.FIELD_TYPE_CHANGED);
```

Comment:
Why is this code removed?

Reply:
You're right, I didn't intend to remove that part 🤦 I've just restored it.